src/lib/modules/blockchain_connector/index.js

```diff
@@ -411,12 +413,24 @@ class BlockchainConnector {
       'get',
       '/embark-api/blockchain/transactions/:hash',
       (req, res) => {
-        self.getTransaction(req.params.hash, (err, transaction) => {
-          if (err) {
-            self.logger.error(err);
-          }
-          res.send(transaction);
-        });
+        const isRawTxHash = req.query.isRawTxHash && req.query.isRawTxHash === 'true';
+
+        if (isRawTxHash) {
+          self.getTransactionByRawTransactionHash(req.params.hash, (err, transaction) => {
+            if (err) {
+              self.logger.error(err);
+            }
+            res.send(transaction);
+          });
+        } else {
+          self.getTransaction(req.params.hash, (err, transaction) => {
+            if (err) {
+              self.logger.error(err);
+            }
+            transaction.raw = self.getRawTransactionHash(transaction);
```

Comment:
This line can be ignored. I just added it for debugging purposes to get a raw transaction hash out of an existing transaction.

Will remove this.